test/katex-spec.js

```diff
@@ -1498,102 +1483,100 @@ describe("A comment parser", function() {
     });
 
     it("should not parse a comment that isn't followed by a newline", () => {
-        expect("x%y").toNotParse();
+        expect`x%y`.not.toParse();
     });
 
     it("should not produce or consume space", () => {
-        expect("\text{hello% comment 1\nworld}")
-            .toParseLike("\text{helloworld}");
-        expect("\text{hello% comment\n\nworld}")
-            .toParseLike("\text{hello world}");
+        expect("\\text{hello% comment 1\nworld}").toParseLike`\text{helloworld}`;
+        expect("\\text{hello% comment\n\nworld}").toParseLike`\text{hello world}`;
```

Comment:
There were some cases of using single backslash with `t` resulting in a horizontal tab(`\t`).